src/renderer/rendererutils.ts

```diff
@@ -20,5 +20,5 @@ const getVideoResult = (video: any): boolean => {
 
 export {
     getFormattedDuration,
-    getVideoResult,
+    getVideoResult
```

Comment:
Pretty please leave trailing commas, so the next addition doesn't show the `getVideoResult` line as having changed when in fact it wasn't :p

Reply:
That's great advice actually. I've never considered this!